modules/installation-configuration-parameters.adoc

```diff
@@ -232,7 +232,7 @@ Required installation configuration parameters are described in the following ta
 |Parameter|Description|Values
```

Comment:
I think this is due to the addition of some super long parameters like `platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKeyServiceAccount`. Is there a way to change the table's wrapping behavior? Otherwise we will need to rethink how we list these deeply nested parameters.

Reply:
I'm unaware of a simple way to do this with adoc. You could play with the various table and cell width options, though you'd need to verify that the table looks good on both the portal and d.o.c. :fearful: